File: cpp/69_kitahara_harukis_gift_433A.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
    int n;
    if (!(cin >> n))
        return 0;
    int c100 = 0, c200 = 0;
    for (int i = 0; i < n; i++)
    {
        int w;
        cin >> w;
        if (w == 100)
            c100++;
        else
            c200++;
    }
    int total = c100 * 100 + c200 * 200;
    if (total % 2 == 0 && (c200 % 2 == 0 || c100 >= 2))
        cout << "YES\n";
    else
        cout << "NO\n";
}

File: python/69_kitahara_harukis_gift_433A.py
n = int(input().strip())
weights = list(map(int, input().split()))
c100 = weights.count(100)
c200 = weights.count(200)
total = c100 * 100 + c200 * 200
if total % 2 == 0 and (c200 % 2 == 0 or c100 >= 2):
    print("YES")
else:
    print("NO")
